test/plugin/test_out_google_cloud.rb

```diff
@@ -428,7 +428,7 @@ def setup
 
   def create_driver(conf = APPLICATION_DEFAULT_CONFIG, tag = 'test')
     Fluent::Test::BufferedOutputTestDriver.new(
-      Fluent::GoogleCloudOutput, tag).configure(conf, use_v1_config: true)
```

Comment:
This is where the .configre function is defined in Fluentd : https://github.com/fluent/fluentd/blob/335d3b2b709ad368da6655636e8ab401b116ae4b/lib/fluent/test/base.rb#L51. Explicitly specifying the variable throws a RuboCop error. The error is `Useless assignment to variable`